src/arch/riscv/kernel/vspace.c

```diff
@@ -105,14 +105,13 @@ BOOT_CODE void map_kernel_frame(paddr_t paddr, pptr_t vaddr, vm_rights_t vm_righ
 
 BOOT_CODE VISIBLE void map_kernel_window(void)
 {
-    /* mapping of KERNEL_ELF_BASE (virtual address) to kernel's
-     * KERNEL_ELF_PHYS_BASE  */
+    /* mapping of physical memory, kernel ELF image and devices. */
```

Comment:
The kernel ELF image also resides in physical memory, and we can only map physical memory or devices, so I don't think this change is an improvement.

The page tables are a mapping from virtual to physical addresses, so the information that we are mapping KERNEL_ELF_BASE to KERNEL_ELF_PHYS_BASE is actually useful information which should not be removed from the comment.